### ISSUE TITLE
Android emulator error reading date and time from network

### ISSUE THREAD

**Issue by lachlanjcollins** [braze-inc/braze-flutter-sdk#74]
I am currently getting a constant feed of error logs when debugging my flutter application using `braze_plugin` v9.0.0 on an Android emulator. The logs spam so repeatedly that it crashes my IDE after running for short period of time (~5 minutes).

Error Log:
```
E/Braze v30.3.0 .DateTimeUtils(32019): Error reading date and time from network. Using system time instead.
E/Braze v30.3.0 .DateTimeUtils(32019): java.time.DateTimeException: Missing network time fix
```

I believe this issue might have been fixed as part of v31.0.0 of the braze-android-sdk as noted here: https://github.com/braze-inc/braze-android-sdk/releases/tag/v31.0.0

**Comment by hokstuff** [comment]
Hi @lachlanjcollins,

Thanks for raising this. We have internally noted to bump the Android SDK bindings for the Flutter SDK. We will ping here when it is released.

**Comment by hokstuff** [comment]
Hi @lachlanjcollins,

We have released [Flutter SDK version 11.0.0](https://github.com/braze-inc/braze-flutter-sdk/releases/tag/11.0.0) which bumps the native Android version to 32.1.0.

Thanks for your patience!
